assets/account-connected/index.html

```diff
@@ -6,11 +6,11 @@
         window.close();
       }, 1);
 
-
       function closeInfoPage() {
-        document.getElementById("close-window").classList.remove("msteams-sync-hidden");
-        document.getElementById("info-window").classList.add("msteams-sync-hidden");
         window.close();
+        document.getElementById("close-window-failed").classList.remove("msteams-sync-hidden");
+        document.getElementById("close-window").classList.add("msteams-sync-hidden");
+        document.getElementById("close-window-icon").classList.add("msteams-sync-hidden");
```

Comment:
Do this after `window.close` so there's no flash if we're successful in closing. Also hide the `X` too.